src/cowboy_swagger.erl

```diff
@@ -6,10 +6,10 @@
 -export([to_json/1, add_definition/1, add_definition/2, add_definition_array/2, schema/1]).
 
 %% Utilities
--export([enc_json/1, dec_json/1]).
+-export([enc_json/1, dec_json/1, normalize_json/1]).
 -export([swagger_paths/1, validate_metadata/1]).
 -export([filter_cowboy_swagger_handler/1]).
--export([get_existing_definitions/2]).
+-export([get_existing_definitions/2, get_global_spec/0, get_global_spec/1]).
```

Comment:
According to Xref, you don't need to export these functions.
I think your plan was to add them as new APIs.
In that case, you need to add them to the list of xref ignores in rebar.config.